src/npm-dependency-version-checker.js

```diff
@@ -32,20 +33,28 @@ class NPMDependencyVersionChecker extends DependencyVersionChecker {
     let basedir = addon.root || getProject(addon).root;
 
     let jsonPath;
+    let key = target + 0x00 + basedir;
```

Comment:
This is another case of what should be \x00, not 0x00, at least if you want it to be the same as hash-for-dep.

Reply:
ya, crap.. :P